### DISCUSSION TITLE
Inserting users in the pb_data.db - tokenKey

### DISCUSSION THREAD

**Discussion by gardc** [pocketbase/pocketbase#2553] [Q&A]
I'm migrating to PB as it seems like a wonderful platform and framework to use! Currently I am in the process of migrating my data from mongoDB to SQLite, so I can insert this SQLite data into the pb_data.db.

Currently I'm having issues with inserting existing users into the users collection, and I get this error: 
`panic: NOT NULL constraint failed: users.tokenKey`. I can't see any documentation on "tokenKey" and the field is not visible when editing the users collection in the UI. What can I do to insert all the existing users?

For information, the users' pashwordHash has been generated in Node.js using bcrypt so it should be compatible AFAIK.

**Comment by hungcrush** [comment]
why don't you use SDK to do that? 

pb.collection("users").create(....) ?

**Comment by ganigeorgiev** [comment]
> I can't see any documentation on "tokenKey" and the field is not visible when editing the users collection in the UI. What can I do to insert all the existing users?

tokenKey is the unique user tokens key part. It must be a non-null **unique random** string (preferably 32+chars).
